CMakeLists.txt

```diff
@@ -42,11 +42,11 @@ include(${MBED_PATH}/tools/cmake/app.cmake)
 add_compile_options(-fdiagnostics-color)
 
 # Include custom targets
-add_subdirectory(${TARGETS_DIR}/TARGET_ARM_REV2)
-add_subdirectory(${TARGETS_DIR}/TARGET_GAMEPAD_REV1)
-add_subdirectory(${TARGETS_DIR}/TARGET_GIMBAL_REV2)
-add_subdirectory(${TARGETS_DIR}/TARGET_PDB_REV2)
-add_subdirectory(${TARGETS_DIR}/TARGET_SCIENCE_REV2)
+add_subdirectory(${TARGETS_DIR}/TARGET_ARM_REV2_2021)
```

Comment:
So would a hypothetical 2022 arm board start at `TARGET_ARM_REV1_2022`? 

suuuuuuuper nit but wouldn't it be better to do `TARGET_ARM_2022_REV1` or even better `TARGET_2022_ARM_REV1`? that way, it groups them together in a file explorer, etc

Reply:
I prefer having the target name as `ARM_REV1_2022` lol

Reply:
After I merge this the github build CI will automatically update the app/target pairs right?

Reply:
U gotta update the yaml. Then u gotta update the github repo settings